### PR TITLE
Review XML CS for the XML files in the component

### PR DESCRIPTION
**Pull request by zero-24**
### Summary of Changes

Review XML CS for the XML files in the component

### Testing Instructions

Confirm the admin & Frontend component options still work (for the frontend it is the submit form)

### Expected result

Use cs

### Actual result

Use not cs

### Documentation Changes Required

None.

### CODE REVIEW

**Comment by astridx** [comment]
I tested this patch succesful.
I applied the patch and created a weblink and changed some configuration. After that I created a menu item and the view in the frontend was correct.
Then I realised, that there are conflics. I was suprised, because it was possible to apply this patch with patch tester. I solved this conflics, I cheated in the patch tester code.
Because of the conflics I am not sure if I can merge this.

**Comment by zero-24** [comment]
@yvesh please take a final review & merge.

Thanks @astridx !

File: src/administrator/components/com_weblinks/config.xml [hunk 1]
```diff
@@ -221,54 +256,63 @@
...
-			</field>
+		<field
+			name="show_cat_num_links_cat"
+			type="radio"
+			class="btn-group btn-group-yesno"
+			default="1"
+			description="COM_WEBLINKS_FIELD_CONFIG_CAT_SHOWNUMBERS_DESC"
+			label="COM_WEBLINKS_FIELD_CONFIG_CAT_SHOWNUMBERS_LABEL"
+		>
+
```
Comment (yvesh) [inline src/administrator/components/com_weblinks/config.xml]: empty line? :-)

**Comment by yvesh** [comment]
@zero-24 isn't there a tab missing before every greater than? :)

https://developer.joomla.org/coding-standards/xml.html

```
<field
	name="count_clicks"
        <!--...-->
>
```

vs

```
<field
	name="count_clicks"
        <!--...-->
	>
```

**Comment by yvesh** [comment]
Closing as #329 fixed it.. Thank you @zero-24 and @astridx

**Comment by astridx** [comment]
Can we delete this because of #329 ?

**Pull request closed without merging**